src/object/player.cpp

```diff
@@ -2090,19 +2083,16 @@ Player::draw(DrawingContext& context)
   */
 
   /* Draw Tux */
-  if (m_safe_timer.started() && size_t(g_game_time * 40) % 2)
+  if (m_safe_timer.started() && size_t(g_game_time * 40) % 2 || (Editor::is_active() || !m_visible))
   {
   }  // don't draw Tux
 
-  else if (m_player_status.bonus[get_id()] == EARTH_BONUS) {
+  else if (m_player_status.bonus[get_id()] == EARTH_BONUS)
+    m_sprite->draw(context.color(), get_pos(), LAYER_OBJECTS + 1);
```

Comment:
Is this check needed? This condition would fall in `else` most of the time, unless `m_dying` is true.

Reply:
it was there when i opened the file so i didnt touch it

Reply:
looking at it, it does look kinda useless. i can remove it